docs/governance/steering/elections.md

```diff
@@ -0,0 +1,129 @@
+
+Restrictions on candidacy and voting are in place to ensure that the CCC project is led by individuals who are highly trusted, impactful, and currently active in the community.
+
+### Eligibility for voting
+
+In order to be eligible for voting, an individual must be:
+
+- A FINOS Member<sup>[1](#footnote1)</sup> who had a substantial contributions 
+  to the CCC project within 12 months prior to their nomination.
+  Contributions include GitHub events like creating issues, creating PRs,
+  reviewing PRs, commenting on issues, etc.
+- An appointed or elected member of one or more active [community groups].
+  within 4 months prior to their nomination.
+
+_It is the responsibility of the steering committee to refine these criteria prior to each election, including setting the number of required contributions, and adding any additional committee memberships that include eligibility._
+
+### Eligibility for candidacy
+
+In order to be eligible for candidacy, an individual must:
+
+* Be eligible for voting in the current Steering Committe election.
+* Accept a nomination or self-nominate.
+* Be endorsed by another party from their own employer.
+* Be endorsed by at least one eligible voter from a financial institution who is
+  not the candidate's employer.
+
+In the event that multiple candidates from the same employer are nominated for a reserved seat, only the candidate with the most votes will be selected.
+
+Refer to the [elections procedure] for more information about how to nominate and endorse candidates in any CCC election.
+
+### Eligibility for Nomination and Endorsement
+
+Any FINOS member may nominate as many people as they wish to. We recommend that you validate the eligibility of your candidate prior to nomination.
+
+[Eligible voters](#eligibility-for-voting) may endorse nominees or campaign as they see fit.
+
+## Terms and Election Cycles
+
+Steering committee members are elected to serve a two (2) year term. Members can serve two (2) consecutive terms and a lifetime of four terms. Terms that result in less than or equal to one (1) year served are exempt.
+
+Election cycles are scheduled such that roughly half of the seats come up for re-election each year for purposes of continuity. The exact number of seats alternates between 3 and 4.
+
+### Bootstrap Election
+
+In the during the [bootstrap] process (which does not follow the guidelines set forth here) 4 seats will be allocated for one (1) year terms, for the purpose of staggering all future elections. The committee will be responsible for self-allocation of these seats.
+
+### Emeritus Term
+
+Members of the steering committee who complete a term will graduate to becoming Emeritus members of the steering committee upon vacating their seat. This confers honor on the recipient, acknowledging the significant contributions they have made to the project. Emeritus members have no binding vote, and no expectation of continued participation in steering committee affairs.
```

Comment:
Is there any other benefit? Or is it just an influential title?

Reply:
Not unless we can think of another benefit to add. This is pretty much copy+paste from the kubernetes emeritus role— I figured it would be good for us to have the same.